packages/web/src/lib/web-mode.ts

```diff
@@ -63,14 +63,18 @@ export class WebMode {
 	}
 
 	/**
-	 * Send transaction to network
-	 * @param signedTxn signed transaction
+	 * Send signed transaction to network and wait for confirmation
+	 * @param signedTxn Signed Transaction blob encoded in base64
 	 */
-	async sendTransaction(signedTxn: any): Promise<JsonPayload> {
-		return await this.algoSigner.send({
+	async sendAndWait(signedTxn: string): Promise<algosdk.modelsv2.PendingTransactionResponse> {
+		const txInfo = await this.algoSigner.send({
 			ledger: this.chainName,
-			tx: signedTxn.blob,
+			tx: signedTxn,
```

Comment:
Algosigner API is change?

Reply:
No, it is same but to keep method consistent with other wallets method in which blob is used, here also the user has to send the blob as parameter rather than the Transaction object